contracts/abstract/JBPayoutRedemptionPaymentTerminal3_1_1.sol

```diff
@@ -1233,12 +1231,7 @@ abstract contract JBPayoutRedemptionPaymentTerminal3_1_1 is
       _split = _splits[_i];
 
       // The amount to send towards the split.
-      uint256 _payoutAmount = _split.percent == leftoverPercentage
-        ? leftoverAmount
-        : PRBMath.mulDiv(_amount, _split.percent, JBConstants.SPLITS_TOTAL_PERCENT);
-
-      // Decrement the leftover percentage.
-      leftoverPercentage -= _split.percent;
+      uint256 _payoutAmount = PRBMath.mulDiv(_amount, _split.percent, _leftoverPercentage);
```

Comment:
Is this correct? isn't it supposed to be:
```
PRBMath.mulDiv(_amount, _split.percent, JBConstants.SPLITS_TOTAL_PERCENT)
```

Reply:
i adapted it to remove a dependency on `leftoverAmount`. i think this works.

Reply:
the denominator _leftoverPercentage decreases as _amount decreases. When _split.percent == _leftoverPercent, only the _amount remains. same as before, just more direct terms. maybe there's a hole in this strategy tho